cmd/satellite/repairer.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/storj/internal/version"
+	"storj.io/storj/pkg/process"
+	"storj.io/storj/pkg/revocation"
+	"storj.io/storj/satellite"
+	"storj.io/storj/satellite/metainfo"
+	"storj.io/storj/satellite/satellitedb"
+)
+
+func cmdRepairerRun(cmd *cobra.Command, args []string) (err error) {
+	ctx, _ := process.Ctx(cmd)
+	log := zap.L()
```

Comment:
Since it's going to be a completely separate process with a completely separate log. I would say probably. But it definitely depends on the devops side.